test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/ModelStateDictionaryTest.cs

```diff
@@ -381,15 +381,19 @@ public void AddModelError_WithErrorString_AddsTooManyModelErrors_WhenMaxErrorsIs
             dictionary.AddModelError("key2", new Exception());
             dictionary.AddModelError("key3", new Exception());
             dictionary.AddModelError("key4", "error4");
-            dictionary.AddModelError("key5", "error5");
-            dictionary.AddModelError("key6", "error6");
 
             // Act and Assert
+            Assert.True(dictionary.CanAddErrors); // though addition will add TooManyModelErrorsException instead.
```

Comment:
not yet.  will of course reverse the assertions and check `MaxErrorsReached` instead.